tmt/utils.py

```diff
@@ -2661,11 +2661,7 @@ def default_branch(repository: Path, logger: tmt.log.Logger,
             return ref.read().strip().split('/')[-1]
 
     # Get the default branch if '.git' is a file
-    command = Command("cat", f"{dot_git}")
-    result = run_command(command=command, cwd=repository, logger=logger)
-    if result.stdout is None:
-        return None
-    *_, worktree_path = result.stdout.strip().split()
+    worktree_path = dot_git.read_text()
```

Comment:
Tests are failing because you omitted the `strip().split()` part.

Reply:
Updated, thanks!